experiments/enableTTL/README.md

```diff
@@ -0,0 +1,188 @@
+# Effect of `enableTTL`
```

Comment:
Just wondering if a 1-2 sentence preamble defining what TTL means is warranted in addition to the description below.

Reply:
Good call. I'm adding: 

> Time-to-live (TTL), in the context of a cache, is a mechanism to indicate how long a resource can exist in the cache before it needs to be refreshed.